activerecord/lib/active_record/encryption.rb

```diff
@@ -53,4 +53,6 @@ def self.eager_load!
       Cipher.eager_load!
     end
   end
+
+  ActiveSupport.run_load_hooks :active_record_encryption, Encryption
```

Comment:
Now that we added this, we need to document in the guides.

Reply:
Done, thanks for the hint.